x/profile/types/genesis_test.go

```diff
@@ -21,65 +21,78 @@ func TestGenesisStateValidate(t *testing.T) {
 			name: "valid custom genesis",
 			genState: &GenesisState{
 				CoordinatorByAddressList: []*CoordinatorByAddress{
-					{CoordinatorId: 10, Address: "spn1c7gh3kejxm3pzl8fwe65665xncs24x5rl7a8sm"},
-					{CoordinatorId: 11, Address: "spn12330zcy9yez37lzrkm6d7fedcu7hc279sgkh3c"},
-					{CoordinatorId: 12, Address: "spn1d6pd5nk08mu789q4msfpynsuha7yf4wcsvvspr"},
-					{CoordinatorId: 13, Address: "spn1ktzsme3g0ag0236ngvkw62vy9tqrr3xysnhp3g"},
+					{CoordinatorId: 0, Address: "spn1c7gh3kejxm3pzl8fwe65665xncs24x5rl7a8sm"},
```

Comment:
Based on the comment above, we must define the value `CoordinatorCount` in each genesis state and put a value below an existing coordinator ID for the test to check this condition